src/types/heartbeat.rs

```diff
@@ -40,7 +47,7 @@ pub struct SignedHeartbeatResponse {
 }
 
 #[derive(Clone, PartialEq, Message)]
-struct ConicalHeartbeat {
+struct CanonicalHeartbeat {
```

Comment:
I assumed `Conical` was a typo here as the other messages use `Canonical`. Hope I was right!

Reply:
Weird, thought I've fixed this. Thanks! You are right.